src/components/PaginationComponent/PaginationComponent.js

```diff
@@ -0,0 +1,36 @@
+import { Pagination, PageItem } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+export default function PaginationComponent({
+  postsPerPage,
+  totalPosts,
+  paginate,
+  currentPage,
+}) {
+  const pageNumbers = [];
+
+  for (let i = 1; i <= Math.ceil(totalPosts / postsPerPage); i += 1) {
```

Comment:
it would be a better practice to change `i+=1` to `i++`

Reply:
The `++` operator was originally used here, but I changed it according to the [no-plusplus rule](https://eslint.org/docs/rules/no-plusplus) and missed the fact that it has this option: `"allowForLoopAfterthoughts": true`. 
But yes, I agree, that in the final expression of a for loop it's better to use unary operators. 
I have already fixed it.